cpg_infra/billing_aggregator/driver.py

```diff
@@ -62,7 +62,7 @@ def main(self):
             return
 
         self.setup_aggregator_functions()
-        self.setup_monthly_export()
+        # self.setup_monthly_export() # we do not need this anymore
```

Comment:
Can you remove all the code for this function to?

```suggestion
```

Reply:
Done, removed bot call and the function.